tsconfig.build.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "include": ["src/index.ts", "css-modules.d.ts"],
```

Comment:
the original config would output files to `lib/src/` (even with `baseUrl` option removed), which we don't want. This achieves the goal of:
- type checking all files, including stories & tests, during development
- but not including stories, tests, or files that are only used in stories/tests in the final build

Lmk if there are concerns or better ways to do this

Reply:
Good thinking

Reply:
aaaand this doesn't include any `Icons` 😅  will fix in the followup when I remove lerna, test publish, etc.